### DISCUSSION TITLE
关于visual studio Intellisense重命名问题

### DISCUSSION THREAD

**Discussion by xkz1994** [BYJRK/DotNet-Discussions#329] [C# 技术交流]
我看up主重命名的时候，使用的是Vs自带的，且有copilot 提示命名
![Clip_2024-08-20_11-05-52](https://github.com/user-attachments/assets/cbf6ac4c-dcf6-4ae4-ba8b-1dc9ce391443)
但是我这边安装了resharper
## 问题：
1. 重命名的时候弹窗是resharper
![Clip_2024-08-20_11-10-14](https://github.com/user-attachments/assets/9dccdd87-57f4-4975-83d4-df85a879c22f)
2. Ctrl+.和Alt+Enter都是调用的resharper的Itellisense 怎么切换为 vs的呢

**Comment by BYJRK** [comment]
我不清楚。我尝试过在 VS 中使用 R#，但是个人感觉体验并不佳。在 VS 中我更倾向于使用 VS 原生的 Intellisense、IntelliCode、Copilot 等功能，而不考虑使用大量由第三方提供的能够明显覆盖掉 VS 默认功能的插件，还比如 Visual Assist（番茄）工具等。我仅在 Rider 中使用 R#

**Reply by xkz1994** [reply]
找到解决方案了，resharper把这个设置勾掉，就是使用的vs的重命名了
![Clip_2024-08-21_10-29-38](https://github.com/user-attachments/assets/e0c91480-f53c-4158-a121-bd21f98d8d5e)

**Reply by xkz1994** [reply]
就可以使用copilot提示命名了
